docs/src/styling.md

```diff
@@ -0,0 +1,112 @@
+# Style guide
+
+The [base16 style guide](https://github.com/chriskempson/base16/blob/main/styling.md)
+is generally targeted towards text editors. Stylix aims to support a variety of
+other applications, and as such it requires its own guide to keep colours
+consistent. Towards this goal we will define several common types of
+applications and how to style each of them using the available colours.
+
+Please keep in mind that this is a general guide; there will be several
+applications that don't fit into any of the groups below. In this case it is up
+to the committer to make sure said application fits in stylistically with the
+rest of the themed applications.
+
+### Terms
+- Alternate: To use when to items need to contrast despite being the same type of item.
+- Item On/Off: This is for on off indicators in applications like a battery charging indicator.
+- List: A list of items to select between such as workspaces in a desktop bar.
+
+## General Applications
+
+General applications are applications that do not fit into any of the more
+specific categories below. This is the scheme you should default to if you are
+unsure. Examples include zathura and sxiv.
+
+- Default Background: base00 
+- Alternate Background: base01
+- Default Text Color: base05
+- Alternate Text Color: base04
+- Selection Color: base03
```

Comment:
Is this the text colour or the background colour?

Reply:
i was thinking of selection as something like the selected item in the sidebar for gtk, maybe we should make it more specific like, sidebar-selection-color?

Reply:
Something like "background of sidebar selection" or "text of sidebar selection" will do